test/common/crypto/import_public_key_corpus/a5cb9723418336b834d6a8cb8d8f772a2fdbe4e7

```diff
@@ -0,0 +1 @@
+?���
```

Comment:
Are all these corpus entries necessary? I wonder if you can merge them to get a smaller corpus with the same coverage.
https://llvm.org/docs/LibFuzzer.html#id24

Reply:
These files have already been merged. 
I ran with libFuzzer for about 20 minutes and I was surprised that there were still 45 files after the merge. That function will invoke another function in openssl, maybe there are a lot of paths inside openssl.